index.html

```diff
@@ -413,55 +413,77 @@ <h3>Proofs</h3>
         </p>
 
         <p>
-A <a>data integrity proof</a> typically includes at least the following
-attributes:
+A <a>data integrity proof</a> can include some of the following attributes:
         </p>
 
         <dl style="margin-left: 1em;">
-          <dt><dfn>type</dfn><dt>
+          <dt><dfn>type</dfn></dt>
           <dd>
-<em>Required</em>. The specific <a>proof type</a> used. For example, an
-<code>Ed25519Signature2020</code> type indicates that the proof includes a
-digital signature produced by an <code>ed25519</code> cryptographic key.
+The specific <a>proof type</a> used for the cryptographic proof MUST be
+specified as a string that maps to a URL [[URL]]. Examples of proof types
+include <code>DataIntegrityProof</code> and <code>Ed25519Signature2020</code>.
+Proof types determine what other mandatory fields are required to secure and
+verify the proof.
```

Comment:
Removed `mandatory` because it is redundant with `required`.

```suggestion
Proof types determine what other fields are required to secure and
verify the proof.
```